### ISSUE TITLE
Map doesn't render at close zoom

### ISSUE THREAD

**Issue by agilob** [osmandapp/Osmand#7174]
After I updated map files today morning I can no longer zoom in on maps closer than `0.25 mi`, map renders grey-ish on zoom 200m and 100m. I tried removing the map file and redownloading it, but didn't help. Map rendering worked last weekend.

OsmAnd~ 3.4.0 from F-Droid
Android 7.1.1

**Comment by vshcherb** [comment]
Please specify what location are using

**Comment by agilob** [comment]
Wales and England

**Comment by xmd5a2** [comment]
I can't reproduce this issue. Please specify exact location.
![Screenshot_2019-07-08-16-35-29-482_net osmand plus](https://user-images.githubusercontent.com/5829014/60814559-859bdd80-a19e-11e9-8189-0d0298084645.png)

**Comment by vshcherb** [comment]
Probably you need to reinstall app, sometimes it happens that update corrupts application, unfortunately I can't reproduce it either (tested in Wales).

**Comment by agilob** [comment]
Ok, I got really disappointed with your proposed solution and started digging if I can backup 100s of my favourites, GPS tracks and custom settings. I couldn't find any option to export my settings safely and reinstall app without losing them.

I started playing more with `Configure map` and found out that checking `Hide` -> `Overground objects` breaks the map rendering. Below two screenshots, first with unchecked hiding overground objects and another with this this flag enabled. This is the same location, the same settings, only one change made.

![Screenshot_20190708-195346](https://user-images.githubusercontent.com/2544251/60835152-572bfb80-a1ba-11e9-8bc3-3e8fe02ff610.png)
![Screenshot_20190708-195356](https://user-images.githubusercontent.com/2544251/60835153-57c49200-a1ba-11e9-9ac3-ae16c9993400.png)
